packages/hooks/package.json

```diff
@@ -17,16 +17,64 @@
     "type": "git",
     "url": "git+https://github.com/Developer-DAO/web3-ui/"
   },
-  "main": "index.js",
   "scripts": {
-    "types": "index.d.ts",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "build": "yarn clean; rollup -c",
```

Comment:
lots of duplication between the components package here, but i mentioned all of that above